doc/nrf/gs_assistant.rst

```diff
@@ -12,6 +12,9 @@ Installing the |NCS| through nRF Connect for Desktop
 On Windows, use the :ref:`gs_app_tcm`.
 On Linux or macOS, use the :ref:`gs_app_gsa`.
 
+.. note::
+   When installing the |NCS| through the |GSA| app, you will not be able to build on the command line using west.
```

Comment:
Should we add a line saying "If you want to build on the command line using west, :ref:`install the nRF Connect SDK manually <gs_installing>`."? Or we don't want to promote the manual installation, as the default one is SES?

Reply:
I was actually thinking about it, but left it out because using SES is our recommendation. I don't know, maybe @ru-fu has an opinion about it too. What do you think?

Reply:
No opinion. The target audience for the GSA installation is very small anyway, and to be honest, I don't know if it doesn't work on Windows only, or if it also doesn't work on Linux/macOS.

Reply:
Ok, we leave it like this for now. We might revisit or tweak it if we get more feedback on this.